projects/1.16/assets/enderio-alloys/enderioalloys/zh_cn.json

```
@@ -0,0 +1,93 @@
+{
+    "itemGroup.enderioalloys": "末影接口丨合金"
+    "block.enderioalloys.block_alloy_conductive_iron": "导电铁块",
+    "item.enderioalloys.item_alloy_nugget_conductive_iron": "导电铁粒",
+    "item.enderioalloys.item_alloy_ingot_conductive_iron": "导电铁锭",
+
+    "block.enderioalloys.block_alloy_construction_alloy": "铁合金块",
+    "item.enderioalloys.item_alloy_nugget_construction_alloy": "铁合金粒",
+    "item.enderioalloys.item_alloy_ingot_construction_alloy": "铁合金锭",
+
+    "block.enderioalloys.block_alloy_crude_steel": "粗钢块",
+    "item.enderioalloys.item_alloy_nugget_crude_steel": "粗钢粒",
+    "item.enderioalloys.item_alloy_ingot_crude_steel": "粗钢锭",
+
+    "block.enderioalloys.block_alloy_crystalline_alloy": "晶化合金块",
+    "item.enderioalloys.item_alloy_nugget_crystalline_alloy": "晶化合金粒",
+    "item.enderioalloys.item_alloy_ingot_crystalline_alloy": "晶化合金锭",
+
+    "block.enderioalloys.block_alloy_crystalline_pink_slime": "晶化粉红粘液块",
+    "item.enderioalloys.item_alloy_nugget_crystalline_pink_slime": "晶化粉红粘液粒",
+    "item.enderioalloys.item_alloy_ingot_crystalline_pink_slime": "晶化粉红粘液锭",
+
+    "block.enderioalloys.block_alloy_dark_steel": "玄钢块",
+    "item.enderioalloys.item_alloy_nugget_dark_steel": "玄钢粒",
+    "item.enderioalloys.item_alloy_ingot_dark_steel": "玄钢锭",
+
+    "block.enderioalloys.block_alloy_electrical_steel": "磁钢块",
+    "item.enderioalloys.item_alloy_nugget_electrical_steel": "磁钢粒",
+    "item.enderioalloys.item_alloy_ingot_electrical_steel": "磁钢锭",

```

```suggestion
    "block.enderioalloys.block_alloy_electrical_steel": "电工钢块",
    "item.enderioalloys.item_alloy_nugget_electrical_steel": "电工钢粒",
    "item.enderioalloys.item_alloy_ingot_electrical_steel": "电工钢锭",
```
我的 1.18 译名草案（未公开）如是说

---

https://github.com/SleepyTrousers/EnderIO/commit/5e724beaebd3202e065db848892ca6bb10823e91
最开始其实是正确的。
我只能说，我佛了